ansible/roles/deploy_helm/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+- name: Label nodes
+  shell: "hyperkube kubectl label nodes {{ item }}=enabled --all"
+  when: deploy_pods_master == true
+  with_items:
+    - "{{ kube_labels }}"
+
+- name: Enable Scheduling of Pods on master
+  shell: "hyperkube kubectl taint nodes {{ ansible_hostname }} dedicated:NoSchedule-"
+  when: deploy_pods_master == true and kube_init | changed
+
+- name: Install Ceph
+  apt:
+    name: ceph-common
+    state: present
+  register: ceph_installed
+
+- name: Create Ceph and OpenStack-Helm directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "/var/lib/openstack-helm/ceph/osd"
+    - "/var/lib/openstack-helm/ceph/ceph"
+    - "/var/lib/openstack-helm/ceph/mon"
+    - "/var/lib/nova/instances"
+
+- name: Install Sigil for Ceph Secrets
+  shell: curl -L https://github.com/gliderlabs/sigil/releases/download/v0.4.0/sigil_0.4.0_Linux_x86_64.tgz | tar -zxC /usr/local/bin
+  when: ceph_installed | changed
+
+- name: Capture kubernetes version
+  shell: kubelet --version | cut -d " " -f2
+  register: kube_version
+
+- name: Deploy kube config
+  template:
+    src: templates/kube-controller-manager.json.j2
+    dest: /etc/kubernetes/manifests/kube-controller-manager.json
+  register: kube_controller_manager
+
+- name: Restart kubelet
+  service:
+    name: kubelet
+    state: restarted
+  when: kube_controller_manager | changed
+
+- action: shell hyperkube kubectl get pods -o wide --all-namespaces | grep kube-controller-manager
+  register: kcm_output
+  until: kcm_output.stdout.find("Running") != -1
+  retries: 20
+  delay: 15
+
+- name: Check for Kubernetes dashboard
+  shell: hyperkube kubectl get pods -o wide --all-namespaces | grep kubernetes-dashboard
+  register: dashboard_check
+  ignore_errors: true
+
+- name: Deploy Kubernetes Dashboard
+  shell: hyperkube kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
+  when: dashboard_check | failed
```

Comment:
do you want dashboard to be an optional element long term as well, or did you want to keep it built into the deployment? may help to have a version included in defaults as well.